### ISSUE TITLE
Steam breaks bspwm

### ISSUE THREAD

**Issue by tilda** [baskerville/bspwm#828]
Hi!

Steam seems to break bspwm with the new chat/friends UI. You can see it in this gif I recorded:
![peek 2018-08-14 19-42](https://user-images.githubusercontent.com/8301262/44127153-a44710d2-a001-11e8-9d0b-7e7ddf04d9fb.gif)

Related issue in Steam's bug tracker: valvesoftware/steam-for-linux#5624

**Comment by gandalf3** [comment]
I haven't had any bspwm-specific problems with it (0.9.5 on arch). I'm not too sure what is happening in the gif. Is the mouse movement intentional?

**Comment by tilda** [comment]
Yes. It shows how the chat breaks the WM (if you click on that area it forces you to the workspace Steam is on)
